src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3195,6 +3193,15 @@ public BoundBlock BindLambdaExpressionAsBlock(ExpressionSyntax body, DiagnosticB
             return bodyBinder.CreateBlockFromExpression(body, bodyBinder.GetDeclaredLocalsForScope(body), refKind, expression, expressionSyntax, diagnostics);
         }
 
+        public BoundBlock BindLambdaExpressionAsBlockContinued(ExpressionSyntax body, BoundExpression expression, DiagnosticBag diagnostics)
```

Comment:
>BindLambdaExpressionAsBlockContinued [](start = 26, length = 36)

"Continued" doesn't really convey anything specific. Continued from where? Etc. Consider renaming with a more specific name, it feels like simply using ```CreateBlockFromExpression``` would work.